packages/client/src/tests/schema.graphql

```diff
@@ -422,6 +432,46 @@ type CertificateSVG {
   status: String
 }
 
+type Question {
+  _id: ID
+  fieldId: String
+  label: MesssageDescriptor
+  placeholder: MesssageDescriptor
+  maxLength: Int
+  fieldName: String
+  fieldType: String
+  preceedingFieldId: String
+  required: Boolean
+  enabled: Boolean
+  custom: Boolean
+  initialValue: String
+}
+
+type formDraft {
+  _id: ID
+  event: String
+  status: String
+  comment: String
+  version: Int
+  history: [DraftHistory]
+  updatedAt: Date
+  createdAt: Date
+}
```

Comment:
Some of the fields here were used as required fields in reducer. So the schema should be in sync with that

Reply:
Yes. Changed.